lib/suspenders/app_builder.rb

```diff
@@ -499,8 +515,108 @@ def setup_default_rake_task
       end
     end
 
+    def users_gems
+      @@user_choice = []
+      choose_template_engine
+      rails_db_gem
+      faker_gem
+      meta_request_gem
+      # Placeholder for other gem additions
+      
+      users_init_commit_choice
+      add_user_gems
+    end
+
+
+    def choose_template_engine
+       variants = {none: 'Erb', slim: 'Slim', haml: 'Haml' }
+       @@user_choice.push  choice 'Select template engine: ', variants
+    end
+
+    def meta_request_gem
+        gem_name = __callee__.to_s.gsub(/_gem/,'')
+        gem_description = <<-TEXT 
+        Rails meta panel in chrome console. Very usefull in AJAX debugging. Save link for chrome add-on.
+        https://chrome.google.com/webstore/detail/railspanel/gjpfobpafnhjhbajcjgccbbdofdckggg
+        TEXT
+       @@user_choice.push  yes_no_question gem_name, gem_description
+    end 
+
+    def rails_db_gem
+       gem_name = __callee__.to_s.gsub(/_gem/,'')
+       gem_description = 'Add gem for pretty view in browser & xls export for models?'
+       @@user_choice.push  yes_no_question gem_name, gem_description
+    end  
+
+    def faker_gem
+       gem_name = __callee__.to_s.gsub(/_gem/,'')
+       gem_description = 'Add gem for generate fake data in testing?'      
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.